tools/integration_tests/operations/operations_test.go

```diff
@@ -107,5 +108,10 @@ func TestMain(m *testing.M) {
 		successCode = only_dir_mounting.RunTests(flags, m)
 	}
 
+	if successCode == 0 {
```

Comment:
Since, we are running the complete integration test three more times. So, integration test runtime would have been 4x. Could you please check, how much time it takes to run the complete integration test right now?

Reply:
Around 32 minutes

Reply:
As we discussed offline, we should skip some non-important tests here:
(a) We can only run the integration test for operations_test package.
(b) We can remove the test case, when both KEY_FILE and ENV_VAR is set. We can cover that under unit test (checking priority of key-file is greater than ENV_VAR) of this method - https://docs.google.com/spreadsheets/d/1ZnMW4zrCTISQBTLmdaDRkbyO1NexVwFXQk5RoQ-hu70/edit?resourcekey=0-K7wXCjQJ4CHhgxPm6q6q2Q#gid=647607290

CC: @sethiay

Reply:
Done